test/dynamo/test_triton_kernels.py

```diff
@@ -958,6 +959,21 @@ def f(x):
         compiled_out = torch.compile(f, fullgraph=True, backend=backend)(x)
         self.assertEqual(compiled_out, eager_out)
 
+    def test_triton_kernel_equal_to_1_arg(self):
```

Comment:
can you add a test with dynamic shapes too?

Reply:
Extended the test to cover dynamic shapes.